### PR TITLE
[stable/openebs]: update openebs charts to 1.11.0 release

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

Update openebs charts to 1.11.0 release
- Updated README
- Updated Chart version
- Updated values.yaml

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/openebs]`)

### CODE REVIEW

**Comment by akhilerm** [comment]
@mittachaitu There are 2 extra columns in `healthCheck.initialDelaySeconds` and `healthCheck.periodSecond`. Can that be removed.

File: charts/openebs/README.md [hunk 1]
```diff
@@ -43,64 +43,63 @@ The following table lists the configurable parameters of the OpenEBS chart and t
 | `image.repository`                      | Specify which docker registry to use          | `quay.io/`                                |
```
Comment (kmova) [inline charts/openebs/README.md:43]: change the default value for image.repository to `""`, and also make this change in the values.yaml. 

As quay.io is going to be under maintenance on 13th and possibly have additional downtime in the coming weeks.

Comment (mittachaitu) [reply charts/openebs/README.md]: Done

**Comment by mittachaitu** [comment]
> initialDelaySeconds

@akhilerm incorporated the comment PTAL.

**Pull request merged**